src/containers/main-page/cv-form/components/fields/projects/components/SkillsToolsDialog.tsx

```diff
@@ -1,86 +1,92 @@
 import { useState, memo } from 'react';
-import { UseFormReturn } from 'react-hook-form';
 
 import {
   DialogActions,
   DialogTitle, DialogContent,
   FormControl, InputLabel, OutlinedInput,
-  MenuItem, SelectChangeEvent, Dialog, Button, Grid,
+  MenuItem, SelectChangeEvent, Dialog, Button, Grid, ListItemText,
 } from '@mui/material';
 
-import { getKeyOf } from 'common/utils/helpers';
-
 import ReactHookFormSelect from 'common/components/react-hook-forms/ReactHookFormSelect';
 import type {
   Category, DbUser, Skill, Tool,
 } from 'common/models/User';
 
-import { ProjectFieldValues } from '../utils/types';
-import { ButtonText, CategoryAddText } from './utils/constants';
+import { ButtonText } from 'common/components/add-pattern/constants';
+import { CategoryAddText } from './utils/constants';
 
+type DialogFormReturn = {
+  category: Category | undefined | null;
+  skill: Skill | undefined | null;
+  tools: string[] | undefined | null;
+};
 interface SkillsToolsDialogProps {
-  formValues: UseFormReturn<ProjectFieldValues>;
   user: DbUser | undefined
   open: boolean;
-  onClose: (event: React.SyntheticEvent<unknown>, reason?: string) => void
+  control: any;
```

Comment:
No any types. Please correct the tsconfig.json and rule for `noImplicitAny: true`
Control has its own typing